app/src/main/res/layout/view_comment_item.xml

```diff
@@ -0,0 +1,63 @@
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/comment_item"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:backgroundTint="?attr/colorBackgroundFloating"
+    android:layout_margin="5dp"
```

Comment:
Can we make this only vertical margin since we're adding horizontal margin to the overall recycler view?